service/models/datasf.py

```diff
@@ -0,0 +1,17 @@
+"""DataSF model"""
+import requests
+
+class DataSF():
```

Comment:
is this general enough that it should be contributed back to the base microservice repo?

Reply:
I think we gonna table this for now. Can pick it up again when we build another microservice will need DataSF integration to align on the use case.